project_app/components/Header.js

```diff
@@ -1,11 +1,33 @@
-import React from "react";
-import NavBarButtons from "./NavBarButtons";
-import Button from "@mui/material/Button";
-import { Container, AppBar, NavBarWrapper, HeaderButtons } from "./styled";
-import Image from "next/image";
-import styled from "styled-components";
+import React from 'react';
+import { useState } from 'react';
+import NavBarButtons from './NavBarButtons';
+import Button from '@mui/material/Button';
+import UserLogin from './UserLogin';
+import Modal from '@mui/material/Modal';
+import Alert from '@mui/material/Alert';
+import Stack from '@mui/material/Stack';
+import { Container, AppBar, NavBarWrapper, HeaderButtons } from './styled';
+import Image from 'next/image';
+import styled from 'styled-components';
 
 export default function Header(props) {
+  const [open, setOpen] = useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+  const [isLoggedOut, setIsLoggedOut] = useState(false);
+
+  const handleLogout = () => {
+    setIsLoggedOut(true);
+  };
+
+  if (isLoggedOut) {
+    return (
+      <Stack sx={{ width: '100%' }} spacing={2}>
+        <Alert severity="success">You have been logged out successfully.</Alert>
+      </Stack>
+    );
+  }
+
```

Comment:
where is the session log out? what if I go to a different page, would I be also logged out there?

Reply:
Added session log out